### PR TITLE
agent release 3.11.0

### PR DESCRIPTION
**Pull request by Abdiramen**

### CODE REVIEW

**Comment by salilsub** [comment]
@Abdiramen is it just when they use the `ngrok http` command, or all http tunnels (including those defined via config)? If the latter, we should clarify that.

**Comment by salilsub** [comment]
@Abdiramen can you also add an item to the "what's new page"? The 3.10 release has an example: https://github.com/ngrok/ngrok-docs/pull/751

**Comment by Abdiramen** [comment]
> Is it just when they use the `ngrok http` command, or all http tunnels (including those defined via config)? If the latter, we should clarify that.

@salilsub It's just `ngrok http` since we don't show logs in the agent for the other two

File: docs/whats-new.mdx [hunk 1]
```diff
@@ -14,6 +14,7 @@ You can expect this page to update regularly (at least monthly). We'll include t
 
 ## June 2024
 
+- 2024-06-13 - Agent now has timestamps for incoming `http` requests.
```
Comment (salilsub) [inline docs/whats-new.mdx:17]: For this one we want to mention the newly released agent version and link to the changelog which has the information on what has actually changed.

Comment (salilsub) [inline docs/whats-new.mdx:17]:
```suggestion
- 2024-06-13 - Released [ngrok Agent 3.10.0](/agent/changelog/#ngrok-agent-3110---2024-06-13).
```

**Review by salilsub — CHANGES_REQUESTED** [review]
One comment about the what's new changes

File: docs/whats-new.mdx (hunk 1 above)
Comment (Abdiramen) [reply docs/whats-new.mdx]: Thanks! I pulled in your chagnes here: https://github.com/ngrok/ngrok-docs/pull/788/commits/4140ed4ac41d99551ecffd0433d972ab5bb407cd

**Review by salilsub — APPROVED** [review]
LGTM

**Pull request merged**